androidlibrary_lib/src/main/java/org/opendatakit/fragment/AlertDialogFragment.java

```diff
@@ -242,19 +252,23 @@ public static AlertDialogFragment eitherReuseOrCreateNew(String alertDialogTag,
       // if no pre-existing fragment create one, else update the message
       if (outputAlertDialogFragment == null) {
          outputAlertDialogFragment = AlertDialogFragment.newInstance(fragmentId, dismissActivity, title, message);
-         if(!outputAlertDialogFragment.isAdded()) {
+         if (!outputAlertDialogFragment.isAdded()) {
             outputAlertDialogFragment.show(fragmentManager, alertDialogTag);
          }
       } else {
          outputAlertDialogFragment.setTitle(title);
          outputAlertDialogFragment.setMessage(message);
+
+
+//Added return statement here
```

Comment:
this return statement is unneeded as it already returns after that by line 267

Reply:
Resolved